packages/core/src/index.ts

```diff
@@ -20,3 +20,4 @@ export {ServerRequest, ServerResponse} from 'http';
 // internals used by unit-tests
 export {parseOperationArgs} from './parser';
 export {ParsedRequest, parseRequestUrl} from './router/SwaggerRouter';
+export {NotFound as HttpErrorNotFound} from 'http-errors';
```

Comment:
IMO, exporting only `NotFound` is too limiting. I am proposing to export full `HttpError` constructor/object.

```ts
// here
export * as HttpError from 'http-errors';

// usage
import {HttpError} from '@loopback/core';
throw new HttpError.NotFound('Current user not found (?!).');
```